### PR TITLE
RBD return output in error case

### PR DESCRIPTION
**Pull request by ghost**
Improves the error from RBD since the output contains the actual errors.

@rootfs

### CODE REVIEW

File: pkg/volume/rbd/rbd_util.go [hunk 1]
```diff
@@ -227,18 +228,19 @@ func (util *RBDUtil) AttachDisk(b rbdBuilder) error {
...
 					[]string{"map", b.Image, "--pool", b.Pool, "--id", b.Id, "-m", mon, "-k", b.Keyring})
 			}
 			if err == nil {
 				break
 			}
+			glog.V(1).Infof("rbd: map error %v %s", err, output)
 		}
 		if err != nil {
-			return err
+			return fmt.Errorf("rbd: map failed %v %v", err, output)
```
Comment (rootfs) [inline pkg/volume/rbd/rbd_util.go]: `output` -> `string(output)` or use `%s`

**Comment by rootfs** [comment]
some nit, rest lgtm, thanks

**Comment by swagiaal** [comment]
@rootfs comment addressed

**Comment by rootfs** [comment]
thanks, lgtm
@markturansky

**Comment by markturansky** [comment]
This is a tidy little improvement.  LGTM.

**Pull request merged**